File: IdeoDigitalApp/ideo-digital/src/app/Models/InvoiceItem.ts
export class InvoiceItem {
    id: number;
    itemName: string;
    quantity: number;
    pricePerUnit: number;
    totalPrice: number;

    constructor() {
        this.id = 0;
        this.itemName = '';
        this.quantity = 0;
        this.pricePerUnit = 0;
        this.totalPrice = 0;
    }
}

File: IdeoDigitalApi/IdeoDigitalApi/Models/InvoiceItem.cs
namespace IdeoDigitalApi.Models
{
    public class InvoiceItem
    {
        public int Id { get; set; }
        public string ItemName { get; set; }
        public int Quantity { get; set; }
        public decimal PricePerUnit { get; set; }
        public decimal TotalPrice { get; set; }
    }
}
